docs/building-a-bento-amp-extension.md

```diff
@@ -92,15 +94,14 @@ In most cases you'll only create the required (req'd) files. If your element doe
 
 ## Extend AMP.PreactBaseElement
 
-Most AMP component extend `AMP.BaseElement`. `AMP.BaseElement` provides hookups and callbacks needed to implement and customize behavior. Callback explanations are in the BaseElement Callbacks section of [Building an AMP Extension](https://github.com/ampproject/amphtml/blob/main/docs/building-an-amp-extension.md), and inlined in the [BaseElement](https://github.com/ampproject/amphtml/blob/main/src/base-element.js)
-class.
+Most AMP components extend `AMP.BaseElement`. `AMP.BaseElement` provides hookups and callbacks needed to implement and customize behavior. Callback explanations are in the BaseElement Callbacks section of [Building an AMP Extension](https://github.com/ampproject/amphtml/blob/main/docs/building-an-amp-extension.md), and inlined in the [BaseElement](https://github.com/ampproject/amphtml/blob/main/src/base-element.js) class.
```

Comment:
Could we forgo a reference to methods in`src/base-element.js`? With auto-envelope, we only have access to a minimal set of methods.

Reply:
Good point, removed. It was originally included to explain that the callbacks are no longer used for those that are familiar with creating classic components:
```
All Preact-based Bento AMP extensions extend `AMP.PreactBaseElement`, which builds upon
`BaseElement` to extricate the component implementation from the aforementioned
callbacks.`
``` 

But I think without being more explicit (i.e. don't use `buildCallback`/`layoutCallback`/etc.), the sentence creates more confusion. Better to only link the resources that _should_ be used, as we do in the "AMP/Preact Bridge" and "PreactBaseElement Callbacks" sections, which it now does. :)